metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala

```diff
@@ -0,0 +1,138 @@
+package scala.meta.internal.decorations
+
+import scala.meta.internal.metals.UserConfiguration
+import scala.meta.internal.metap.PrinterSymtab
+import scala.meta.internal.semanticdb.Print
+import scala.meta.internal.{semanticdb => s}
+import scala.meta.metap.Format
+
+object SemanticdbTreePrinter {
```

Comment:
Super random, but I happened to look at the git blame for this file and it shows that this was added 2019-11-30? Why is it showing that?

Reply:
<img width="999" alt="Screenshot 2020-10-01 at 11 23 13" src="https://user-images.githubusercontent.com/13974112/94791877-84288780-03d8-11eb-8cb6-1f12dad3ee6e.png">

Actually I think somehow your force push is showing up as last year?

Reply:
haha, that's when I started looking into it :P so actually it's kind of right. Maybe I overdid it with the amends though.

Reply:
O wow haha, ok. I might be best to ammend the commit date to better reflect _this year_ 😆

Reply:
Amended it with current dat, thanks for noticing!